dmscripts/bulk_upload_documents.py

```diff
@@ -0,0 +1,39 @@
+import os
+import re
+import sys
+sys.path.insert(0, '.')
+from dmutils.documents import get_countersigned_agreement_document_path
```

Comment:
It looks like a recent change to utils has removed the `get_countersigned_agreement_document_path` method altogether :/ - does it need to be put back?  Or is there a new method that can be used here that will do the job?